test/common/quic/envoy_quic_client_session_test.cc

```diff
@@ -133,15 +133,17 @@ class EnvoyQuicClientSessionTest : public testing::TestWithParam<bool> {
     setQuicConfigWithDefaultValues(envoy_quic_session_.config());
     envoy_quic_session_.OnConfigNegotiated();
     envoy_quic_session_.addConnectionCallbacks(network_connection_callbacks_);
-    envoy_quic_session_.setConnectionStats(
-        {read_total_, read_current_, write_total_, write_current_, nullptr, nullptr});
+    envoy_quic_session_.setConnectionStats({read_total_, read_current_, write_total_,
+                                            write_current_, nullptr, nullptr,
+                                            &self_connection_close_error_, nullptr});
     EXPECT_EQ(&read_total_, &quic_connection_->connectionStats().read_total_);
   }
 
   void TearDown() override {
     if (quic_connection_->connected()) {
       EXPECT_CALL(*quic_connection_,
                   SendConnectionClosePacket(quic::QUIC_NO_ERROR, _, "Closed by application"));
+      EXPECT_CALL(self_connection_close_error_, recordValue(0));
```

Comment:
Are there any places where we might be able to EXPECT_CALL a non-zero value? We may well not have any such tests at the envoy level?

Reply:
Right now we don't. But if more tests are added in this file in the future, I expect we will have recordValue() with non-zero parameter.